RileyLinkBLEKit/RileyLinkBLEDevice.m

```diff
@@ -278,7 +278,7 @@ - (void)checkVersion {
         NSString *foundVersion;
         BOOL versionOK = NO;
         // We run two commands here, to flush out responses to any old commands
-        [s doCmd:cmd withTimeoutMs:1000];
+        [s doCmd:cmd withTimeoutMs:5000];
```

Comment:
Gotcha. I thought I remembered this failing only when there's a mismatch in the command-response conversation because of a sudden re-launch of the app. I didn't think a timeout could help that issue, but you're right that it can't hurt either.